sdk/keyvault/azure-security-keyvault-administration/src/main/java/com/azure/security/keyvault/administration/implementation/models/RoleAssignment.java

```diff
@@ -87,9 +87,6 @@ public RoleAssignment setProperties(RoleAssignmentPropertiesWithScope properties
     @Override
     public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
         jsonWriter.writeStartObject();
-        jsonWriter.writeStringField("id", this.id);
-        jsonWriter.writeStringField("name", this.name);
-        jsonWriter.writeStringField("type", this.type);
```

Comment:
This is sus.

Reply:
This was a change in code generation logic to stop serializing response only values, Effectively, this makes this the same as how Jackson Databind would have handled serialization.

Reply:
Verified it's not needed to create role assignments (or defs below). Thanks for explaining!